client/ts/drawer.ts

```diff
@@ -0,0 +1,87 @@
+
+
+
+export class Drawer {
```

Comment:
could you add some docstring to explain what the Drawer is doing?

Reply:
No way.
Never gonna happen.

Reply:
Okay... done.

Reply:
😈